libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1079,37 +1079,17 @@ bool RC_Channel::do_aux_function(const aux_func_t ch_option, const AuxSwitchPos
         break;
 
     case AUX_FUNC::EKF_POS_SOURCE:
-        switch (ch_flag) {
-        case AuxSwitchPos::LOW:
-            // low switches to primary source
-            AP::ahrs().set_posvelyaw_source_set(0);
-            break;
-        case AuxSwitchPos::MIDDLE:
-            // middle switches to secondary source
-            AP::ahrs().set_posvelyaw_source_set(1);
-            break;
-        case AuxSwitchPos::HIGH:
-            // high switches to tertiary source
-            AP::ahrs().set_posvelyaw_source_set(2);
-            break;
-        }
+        // LOW(0)=primary, MIDDLE(1)=secondary, HIGH(2)=tertiary
+        AP::ahrs().set_posvelyaw_source_set((uint8_t)ch_flag);
```

Comment:
To do this we'd want a comment where the `AuxSwitchPos` enumeration is created to indicate some users cast directly to a `uint8_t`, so the values have to be considered fixed.  We would also want to nominate the values in that enumeration rather than just leaving them at defaults.

Reply:
Thanks for the review.
OK, I will leave them at original.
When using enumeration, there are many changes. (check "source_set_idx" from source code)

Reply:
I thought I should define enum class.
but enum is enough for this, right?